gossip/src/crds.rs

```diff
@@ -56,6 +57,22 @@ use {
 const CRDS_SHARDS_BITS: u32 = 12;
 // Number of vote slots to track in an lru-cache for metrics.
 const VOTE_SLOTS_METRICS_CAP: usize = 100;
+// Required number of trailing zero bits for crds signature to get reported to influx
+// mean new push messages received per minute per node
+//      testnet: ~500k,
+//      mainnet: ~280k
+// target: 1-2 signatures reported per minute
+// log2(250k) = ~17.9.
+const SIGNATURE_SAMPLE_TRAILING_ZEROS: u32 = 18;
+
+/// check if last SIGNATURE_SAMPLE_TRAILING_ZEROS bits of signature are 0
+fn should_report_message_signature(signature_bytes: Signature) -> bool {
```

Comment:
The argument is no longer `signature_bytes`; it is just `signature`. Also we probably want this to be `&Signature` not to copy 64 bytes each time.

also to keep this file organized, can we move this helper function to the end after all structs, right before `mod tests`?

also might be good to add `#[inline]` annotation to this function since it is invoked for every signature but only in one place.